modules/installation-creating-azure-control-plane.adoc

```diff
@@ -64,10 +64,12 @@ ifdef::azure[]
 $ az deployment group create -g ${RESOURCE_GROUP} \
   --template-file "<installation_directory>/05_masters.json" \
   --parameters masterIgnition="${MASTER_IGNITION}" \ <1>
-  --parameters baseName="${INFRA_ID}" <2>
+  --parameters baseName="${INFRA_ID}" \ <2>
+  --parameters masterVMSize="Standard_D8s_v3" <3>
```

Comment:
Is there a reason why this uses a value of `Standard_D8s_v3` while the other two sections have `Standard_D4s_v3`?

Reply:
That's what we use in the Installer templates and in IPI: https://github.com/openshift/installer/blob/master/docs/user/azure/limits.md#instance-limits. See also https://github.com/openshift/installer/blob/master/pkg/types/azure/defaults/machines.go#L10-L17